src/components/Cover/Cover.module.css

```diff
@@ -0,0 +1,95 @@
+.wrapper {
+    text-decoration: none;
+    min-height: 200px;
+    overflow: auto;
+    display: grid;
+    margin-top: 66px;
+    opacity: 1;
+}
+
+.rectangle {
+    grid-row: 1;
+    grid-column: 1;
+    opacity: 0.3;
+    background-color: #2A2C2F;
+    z-index: 1;
+}
+
+.wrapper:hover .rectangle {
+    transition: opacity .4s;
+    opacity: 0.8;
+}
+
+.coverPhoto {
+    grid-row: 1;
+    grid-column: 1;
+width: 100%;
+height: 100%;
+z-index:0;
+}
+
+.text {
+    grid-row: 1;
+    grid-column: 1;
+    z-index: 2;
+}
+
+.title {
+    max-width: 238px;
+    margin: 33px auto 0 auto;
+    color: white;
+    text-align: center;
+    font-family: "InterBlack";
+    font-size: 32px;
+    line-height: 35px;
+}
+
+.subtitle {
+    margin: 16px auto 0 auto;
+    min-width: 238px;
+    color: white;
+    text-align:center;
+    font-family: "InterRegular";
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: normal;
+}
+
+@media (min-width: 768px) {
+    .wrapper {
+        min-height: 480px;
```

Comment:
I do not see why we need it with the margins inside. If you will remove them you might have it, but this margins the element knows its height

Reply:
removed `min-height`